keyboards/flutterby/keymaps/default/keymap.c

```diff
@@ -0,0 +1,161 @@
+#include "flutterby.h"
```

Comment:
Please change this to: 

```suggestion
#include QMK_KEYBOARD_H
```